NARFO_BE/NARFO_BE/Models/User.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NARFO_BE.Models
+{
+    public partial class User
+    {
+        public int UserId { get; set; }
```

Comment:
i would recommend when making an id on a table always use a long if it is going to be used as a primary key